Gruntfile.js

```diff
@@ -70,9 +70,16 @@ module.exports = function(grunt) {
             }
         },
 
-        jsFiles: [
+        jsHeadFiles: [
             // Include:
+            'source/js/head-scripts.js'
+        ],
+
+        jsFootFiles: [
+            // Include:
+            'source/js/jquery-2.0.0b2.js',
```

Comment:
It's up to you, but you'll get a slight performance gain my keeping it all together. If you take it out, it's one more http request and an additional file for the browser to cache.